nuget/spec/dependabot/nuget/native_helpers_spec.rb

```diff
@@ -192,4 +192,67 @@
       end
     end
   end
+
+  describe "#ensure_no_errors" do
```

Comment:
Previously we didn't have unit tests for converting updater errors into Ruby errors; this fixes that for all cases.